go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/codeready-toolchain/api v0.0.0-20200323155710-2cad93f41d50
 	github.com/codeready-toolchain/toolchain-common v0.0.0-20200330115326-55526af245bd
 	github.com/go-logr/logr v0.1.0
+	github.com/golangci/golangci-lint v1.24.0 // indirect
```

Comment:
(this happened because I ran `make lint` locally before pushing to GitHub and opening this PR.)